src/Analysis/Ast/Impl/Diagnostics/ErrorCodes.cs

```diff
@@ -25,5 +25,6 @@ public static class ErrorCodes {
         public const string UndefinedVariable = "undefined-variable";
         public const string VariableNotDefinedGlobally= "variable-not-defined-globally";
         public const string VariableNotDefinedNonLocal = "variable-not-defined-nonlocal";
+        public const string NewTypeArguments = "newtype-arguments";
```

Comment:
I think we should prefix all of these, so do `typing-newtype-arguments` instead (and rename the constant to match).